bin/alea-submission

```diff
@@ -10,12 +10,26 @@ def main():
         "--computation",
         type=str,
         required=True,
-        choices=[
-            "discovery_power",
-            "threshold",
-            "sensitivity",
-        ],
-        help="Type of computation, defined in YAML runner config file",
+        help=(
```

Comment:
my IDE complains about """  block quotes, but the solution here seems way worse for readability surely?

Reply:
Ah got it. You want the names to have "" -- then I think this is fine.